ani-cli

```diff
@@ -465,8 +498,13 @@ check_input
 append_history
 open_selection
 
+
+########
+# Loop #
+########
+
 while :; do
-	printf "\n${c_green}Currently playing %s episode ${c_cyan}%d/%d\n" "$selection_id" "$episode" "$last_ep_number"
+	inf "Currently playing $selection_id episode $episode/$last_ep_number"
```

Comment:
I think the sacrifice in UI prettiness is worth the gain in uniformity. You can leave this as is